docs/user/tutorials/quickstart.md

```diff
@@ -0,0 +1,138 @@
+# Quickstart
+
+This guide covers how you can quickly get started using Hypper.
+
+If you already know Helm, this will be a breeze, as Hypper follows Helm's UI,
+being a drop-in replacement, and extends its functionalities to install Helm
+and Hypper charts alike.
+
+## Prerequisites
+
+The following prerequisites are required for a successful and properly secured
+use of Hypper.
+
+1. A Kubernetes cluster.
+1. Deciding what security configurations to apply to your installation, if any
+1. Installing and configuring Hypper.
```

Comment:
I don't think we need to talk about securing a cluster. People may just kick the tires in something like Docker Desktop. All they need is a Kubernetes cluster, Hypper, and a chart to install.